prow/plugins/trigger/generic-comment.go

```diff
@@ -156,13 +157,16 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 						"repo":    repo,
 					})
 					runID := run.ID
-					go func() {
-						if err := c.GitHubClient.TriggerFailedGitHubWorkflow(org, repo, runID); err != nil {
-							log.Errorf("attempt to trigger github run failed: %v", err)
-						} else {
-							log.Infof("successfully triggered action run")
+					if err := c.GitHubClient.TriggerFailedGitHubWorkflow(org, repo, runID); err != nil {
+						log.Errorf("attempt to trigger github run failed: %v", err)
+						if strings.Contains(err.Error(), "Unable to retry this workflow run because it was created over a month ago") {
+							resp := "The workflow triggered has expired. You can find more information about the same here <>"
```

Comment:
Let's also include the name of the expired workflow in the comment here, you can grab it from `run.Name`.